mcwrangler.js

```diff
@@ -129,84 +95,126 @@ const findCloudflareTargets = (inDir) => {
   return targets;
 }
 
+
+// Find which routes correspond to which worker on which environment.
+const buildWorkerRoutes = (envs, workersJSON) => {
+  let workersRoutes = {};
+  for(const [envName, env] of Object.entries(envs)){
+    let addEntry = (pattern, script_name) => {
+      if(workersRoutes[script_name] === undefined) 
+        workersRoutes[script_name] = {};
+      if(workersRoutes[script_name][envName] === undefined) 
+        workersRoutes[script_name][envName] = [];
+      workersRoutes[script_name][envName].push(pattern);
+    };
+  
+    // Substitute in per-target host aliases to make the final list of routes
+    workersJSON.worker_routes.forEach((workerRoute)=> {
+      let pattern = workerRoute.pattern;
+      let slash_index = pattern.indexOf("/") ?? pattern.length;
+      let substitution = (env.aliases ?? []).find((value) => {
+        return value.alias.length === slash_index && pattern.slice(0, slash_index) === value.alias
+      })
+      if (substitution === undefined) {
+        addEntry(pattern, workerRoute.script_name);
+      } else {
+        let suffix = pattern.slice(slash_index);
+        for (const host of substitution.host) {
+          addEntry(host + suffix, workerRoute.script_name);
+        }
+      }
+    });
+  }
+  return workersRoutes;
+}
+
+// Write out a configuration for a single worker.
+const processWorker = (workerName, logpush, allRoutes, targetName, unpackDirName, envs, existingWorkers) => {
+  const outputHead = `
+name = "${workerName}"
+
+workers_dev = true
+compatibility_date = "2023-12-01"
+main = "worker.js"
+logpush = ${logpush}
```

Comment:
can add the following, which is added in Darwinium edge publisher:
`minify = true`